libs/@local/hash-isomorphic-utils/src/generate-entity-label.ts

```diff
@@ -111,7 +111,9 @@ export const generateEntityLabel = (
   const options = [
     "name",
     "preferredName",
+    "preferred-name", // @todo H-2561 account for proposed entities having non-standard property keys now
     "displayName",
+    "display-name", // @todo H-2561 account for proposed entities having non-standard property keys now
```

Comment:
Are these todos and the task still relevant since https://github.com/hashintel/hash/pull/4346 merged? The proposed entities logged to the frontend will now be converted to have Base URLs again.

The `ProposedEntity` outputted by actions should have properties defined via base URLs in general, the simplified properties should only be relevant for LLM calls.

Reply:
Nope – removed in https://github.com/hashintel/hash/pull/4357/commits/65436ab3ec05284f449a37d6abddc6bf434ed401